tests/fileTests.py

```diff
@@ -32,7 +32,7 @@ def setUpClass(cls):
 
         # setup submission/jobs data for test_check_status
         cls.status_check_submission_id = cls.insertSubmission(
-            cls.jobTracker, cls.submission_user_id, agency = "Department of the Treasury", startDate = "04/2016", endDate = "04/2016")
+            cls.jobTracker, cls.submission_user_id, agency = "Department of the Treasury", startDate = "10/2015", endDate = "06/2016", is_quarter = True)
```

Comment:
Ooh, nice. I wouldn't have thought to test across across as well as testing incorrect months.